packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -320,6 +320,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
               language: { translate: t, locale: booking.user?.locale ?? "en" },
             },
             attendees,
+            location: bookingMetadataSchema.parse(booking.metadata || {})?.videoCallUrl || booking.location,
```

Comment:
Given that the title works properly in the email content where it uses `reminder.booking.eventType?.title`, I wonder if the following would fix the title issue:

```suggestion
            location: bookingMetadataSchema.parse(booking.metadata || {})?.videoCallUrl || booking.location,
            title: booking.title || booking.eventType?.title || "",
```

Reply:
Yeah I was thinking the same. Let's do it and check